zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -491,9 +491,11 @@ function ProjectDetails(props) {
           aria-labelledby={t('projectDetails.ariaLabels.deleteProject')}
         >
           <DialogTitle id="delete-project">
+            <Typography variant="h4">
             {t('projectDetails.project.delete.dialog.primary')}
+            </Typography>
           </DialogTitle>
-          <Box
+          {delete_project_dialog_error !== null &&<Box
             component="p"
             className={delete_project_dialog_error !== null && classes.errorBox}
```

Comment:
since `delete_project_dialog_error !== null &&` was moved so we can conditionally remove the <Box/> component where there is no error, it serves no other purpose there. It makes sense to remove it.